src/components/LanguagePicker/LanguagePickerListItem.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {StyleSheet, TouchableOpacity} from 'react-native'
+
+import {Icon} from '../Icon'
+import {Text} from '../Text'
+
+type Props = {
+  text: string
```

Comment:
language

Reply:
Component removed. It will be applied next time. Thanks for the suggestions!